### PR TITLE
Feature/testbed of testbeds

### PR DESCRIPTION
**Pull request by mbwatson**
this PR adds Testbed of Testbeds feature.

- testbed data lives in `src/data/resources/testbeds.yaml`
- testbed details are rendered at `/resources/testbeds`
- a new menu item is added, RESOURCES > TESTBEDS

### CODE REVIEW

File: src/data/resources/testbeds/testbeds.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    Chameleon is a large-scale, deeply reconfigurable experimental platform built to support Computer Sciences systems research.  
+
+- name: CloudLab
+  url: https://www.cloudlab.us/
+  image: ./cloudlab.png
+  description: >
+    Flexible, scientific infrastructure for research on the future of cloud computing. Researchers use CloudLab to build their own clouds, experimenting with new architectures that will form the basis for the next generation of computing platforms. 
+
+- name: Cosmos
+  url: https://powderwireless.net/
```
Comment (yaxue1123) [inline src/data/resources/testbeds/testbeds.yaml:15]: The link should be: https://www.cosmos-lab.org/

File: src/pages/resources/testbeds.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+      <div className="header" style={{ backgroundImage: `url(${ testbed.image.childImageSharp.original.src })` }}>
+        <Subheading center hidden>{ testbed.name }</Subheading>
+      </div>
+      <div className="description">
+        <Paragraph>
+          { testbed.description }
+        </Paragraph>
+      </div>
+      <div className="footer">
+        <a href={ testbed.url } rel="noopener noreferrer" className="testbed-link" aria-label={ `Visit the ${ testbed.name } website` }>
```
Comment (yaxue1123) [inline src/pages/resources/testbeds.js:68]: Would adding "target=_blank" to the <a> be better? In this way users can open multiple testbeds' website with multiple tabs easier.

**Review by yaxue1123 — COMMENTED** [review]
The new page looks great! and the code is succinct. 👍

**Comment by mbwatson** [comment]
@yaxue1123 thanks! i'll probably request one more round of review after i push the most recent changes and it gets a final look from the team.

**Comment by mbwatson** [comment]
@yaxue1123 let me know if you have any ideas for improving this new page or its content.

the preview deploy is here: https://deploy-preview-168--friendly-allen-014775.netlify.app/resources/testbeds/

File: src/pages/resources/testbeds.js (hunk 2 above)
Comment (yaxue1123) [reply src/pages/resources/testbeds.js:77]: Also, is it possible to make the whole footer clickable as the link? I was confused at first by clicking the footer but not being directed to a new page. The footer changes color by hovering on it, which may make users think the whole footer is clickable as a link button.

Comment (mbwatson) [reply src/pages/resources/testbeds.js]: yep, i totally forgot the target attr. nice catch! regarding the footer, i agree. does this new implementation feel right?

File: src/data/resources/testbeds/testbeds.yaml (hunk 1 above)
Comment (mbwatson) [reply src/data/resources/testbeds/testbeds.yaml:15]: thanks for catching that!

File: src/pages/resources/testbeds.js (hunk 2 above)
Comment (yaxue1123) [reply src/pages/resources/testbeds.js]: Yes the new implementation looks great!

**Pull request merged**